pkg/tcl/testworkflowstcl/testworkflowcontroller/logs.go

```diff
@@ -39,9 +39,13 @@ func (i *Instruction) ToInternal() *testkube.TestWorkflowOutput {
 	if i == nil {
 		return nil
 	}
-	value := &i.Value
-	if i.Value == nil {
-		value = nil
+	value := map[string]interface{}(nil)
+	if i.Value != nil {
+		v, _ := json.Marshal(i.Value)
+		_ = json.Unmarshal(v, &value)
```

Comment:
should we add a warning when there are errors?

Reply:
We could add one, good point :+1: